### PR TITLE
Add clients

### PR DESCRIPTION
**Pull request by daniejoh**
Adds all clients
Adds tests for all clients

### CODE REVIEW

File: src/main/java/io/openepi/Main.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package io.openepi;
+
+//TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
+// click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
+public class Main {
```
Comment (kstigen) [inline src/main/java/io/openepi/Main.java:5]: Looks like we should delete this Main-file

File: src/main/java/io/openepi/weather/api/WeatherApi.java [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+     </table>
+     */
+    public okhttp3.Call getSunriseSunriseGetAsync(BigDecimal lat, BigDecimal lon, String date, final ApiCallback<METJSONSunrise> _callback) throws ApiException {
+
+        okhttp3.Call localVarCall = getSunriseSunriseGetValidateBeforeCall(lat, lon, date, _callback);
+        Type localVarReturnType = new TypeToken<METJSONSunrise>(){}.getType();
+        localVarApiClient.executeAsync(localVarCall, localVarReturnType, _callback);
+        return localVarCall;
+    }
+}
```
Comment (kstigen) [inline src/main/java/io/openepi/weather/api/WeatherApi.java:380]: As a general note, is it possible to make the public interface of this (and the other clients) easier for end users?

Comment (daniejoh) [reply src/main/java/io/openepi/weather/api/WeatherApi.java:380]: Yes, a lot can be improved.

I believe there are several improvements that can be done. However, each change we do strays further away from the generated code and makes it harder to just "copy" it if we make a change to the APIs later. In other words, more to maintain. But I definitely see the need to make them "simpler". I think it should be a separate PR though.

Comment (kstigen) [reply src/main/java/io/openepi/weather/api/WeatherApi.java:380]: Agree to do this in a separate PR. 
The reason for doing this is that I think that a designated client library should create more value for the user, than just allowing the user to skip doing the client generation themselves.

Comment (daniejoh) [reply src/main/java/io/openepi/weather/api/WeatherApi.java:380]: Agree on that 😄

**Review by kstigen — APPROVED** [review]

**Pull request merged**